core/execution/reference_price_moves.go

```diff
@@ -50,6 +50,9 @@ func (m *Market) checkForReferenceMoves(
 		return
 	}
 
+	// will be set to non-nil if a peg is missing
+	_, _, err := m.getBestStaticPricesDecimal()
```

Comment:
Why not get the errors on the following lines, en stop early?

Reply:
because we would need to check 4 errors, and we still need to get the 4 relevant values to update the market fields and set `stateChanged = true`. By checking this error once, I didn't need to change any existing code, just add the error check in there. I thought about doing something with the errors returned by the existing calls, but it ended up being more verbose anyway

Reply:
ah right... stateChanged is unused though anyway now